motor.h

```diff
@@ -2,6 +2,8 @@
 #define MOTOR_H
 
 #include "Arduino.h"
+#define FORWARD 0
+#define BACKWARD 1
```

Comment:
Wouldn't it be better to prefix those constants with `MOTOR_` for example? As to distinguish from other `FORWARD` or `BACKWARD` constants that might be used in the future.